package.json

```
@@ -38,9 +39,10 @@
     "eslint-plugin-sql-template": "^2.0.0"
   },
   "devDependencies": {
-    "@uphold/github-changelog-generator": "^1.0.1",
-    "eslint": "^7.18.0",
-    "mocha": "^7.1.1",
+    "@uphold/github-changelog-generator": "^3.0.0",
+    "babel-eslint": "^10.1.0",
```

```suggestion
    "babel-eslint": "^10.1.0",
```
Why this one is needed?

---

Actually I can remove that now. The `requireConfigFile: false` option negates it. Good spot.